src/CalcManager/Ratpack/ratconst.h

```diff
@@ -1,480 +1,480 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
```

Comment:
Not related to your change but, since you're here, can you add `#pragma once`?

Reply:
Sure, will do!